packages/dd-trace/src/plugins/database.js

```diff
@@ -4,6 +4,61 @@ const StoragePlugin = require('./storage')
 
 class DatabasePlugin extends StoragePlugin {
   static get operation () { return 'query' }
+
+  constructor (...args) {
+    super(...args)
+    this.serviceTags = {
+      dddbs: '',
+      encodedDddbs: '',
+      dde: '',
+      encodedDde: '',
+      ddps: '',
+      encodedDdps: '',
+      ddpv: '',
+      encodedDdpv: ''
+    }
+  }
+  propagationHelper (serviceTag, encodeATag, spanConfig) {
+    if (serviceTag !== spanConfig) {
+      this.serviceTags[serviceTag] = spanConfig
+      this.serviceTags[encodeATag] = encodeURIComponent(spanConfig)
+    }
+  }
+
+  createDBMPropagationCommentService () {
+    const local = this.serviceTags
+    this.propagationHelper('dddbs', 'encodedDddbs', this.config.service)
+    this.propagationHelper('dde', 'encodedDde', this.tracer._env)
+    this.propagationHelper('ddps', 'encodedDdps', this.tracer._service)
+    this.propagationHelper('ddpv', 'encodedDdpv', this.tracer._version)
+
+    return `dddbs='${local.encodedDddbs}',dde='${local.encodedDde}',` +
+    `ddps='${local.encodedDdps}',ddpv='${local.encodedDdpv}'`
+  }
+
+  // To Do create the buildTraceparent as a helper function function possibly in trace
```

Comment:
Since it's just a matter of putting this function elsewhere, let's do it in this PR to avoid code duplication.

Reply:
Sure thing! Still not sure where @rochdev @bengl @Qard @tlhunter let me know where I can add this function. Thomas suggested previously creating a file in `datadog-core`, such as
`packages/datadog-core/src/traceparent.js`. Let me know if this is a good spot.

Reply:
Yeah that's fine. It can always be moved later if we need, I just want to avoid the duplicated code.

Reply:
Maybe we could put it on SpanContext as `toTraceparent`, similar to how we currently have `toTraceId` and `toSpanId`?

Reply:
It's worth noting that these 2 methods are from OpenTracing and if we add methods we're deviating from the spec. This type of change will be easier when the API is decoupled from actual implementation.